<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsdWidget</class>
 <widget class="QFrame" name="OsdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>90</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="VlcSeekWidget" name="seek" native="true"/>
   </item>
   <item row="3" column="2">
    <widget class="InfoBarWidget" name="info" native="true"/>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QLabel" name="logo"/>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="number">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>48</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="numDigits">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPlay">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Play</string>
       </property>
       <property name="statusTip">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/48x48/media-playback-start.png</normaloff>:/icons/48x48/media-playback-start.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStop">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Stop</string>
       </property>
       <property name="statusTip">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/48x48/media-playback-stop.png</normaloff>:/icons/48x48/media-playback-stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Back</string>
       </property>
       <property name="statusTip">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/48x48/media-skip-backward.png</normaloff>:/icons/48x48/media-skip-backward.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Next</string>
       </property>
       <property name="statusTip">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/48x48/media-skip-forward.png</normaloff>:/icons/48x48/media-skip-forward.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRecordNow">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Record Now</string>
       </property>
       <property name="statusTip">
        <string>Record Now</string>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/48x48/media-record.png</normaloff>:/icons/48x48/media-record.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTeletext">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Teletext</string>
       </property>
       <property name="statusTip">
        <string>Teletext</string>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/48x48/teletext.png</normaloff>:/icons/48x48/teletext.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="teletext">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>48</height>
        </size>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>888</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMute">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Mute</string>
       </property>
       <property name="statusTip">
        <string>Mute</string>
       </property>
       <property name="icon">
        <iconset resource="../../../data/icons.qrc">
         <normaloff>:/icons/48x48/volume-muted.png</normaloff>:/icons/48x48/volume-muted.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VlcVolumeSlider" name="volume" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcVolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">vlc-qt/VolumeSlider.h</header>
   <container>1</container>
   <slots>
    <slot>mute()</slot>
    <slot>volumeUp()</slot>
    <slot>volumeDown()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VlcSeekWidget</class>
   <extends>QWidget</extends>
   <header location="global">vlc-qt/SeekWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoBarWidget</class>
   <extends>QWidget</extends>
   <header>ui/core/InfoBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonTeletext</sender>
   <signal>toggled(bool)</signal>
   <receiver>teletext</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
